public/doc/swagger-2.yaml

```diff
@@ -767,35 +773,35 @@ definitions:
         readOnly: true
       processed_by:
         type: string
-        description: Person to process the action
+        description: Processor info who perform the action
       actioned_at:
         type: string
         format: date-time
-        description: Timestamp of action be processed
+        description: Timestamp of action processed
```

Comment:
Action no longer has `actioned_at` and `notified_at` timestamp. Since it is created once and read-only. The creation time stamp is the action time.